README.md

```diff
@@ -1,24 +1,101 @@
-# twingly-analytics-api-java
+# twingly-search-api-java [![Build Status](https://travis-ci.org/xSAVIKx/twingly-search-api-java.svg?branch=master)](https://travis-ci.org/xSAVIKx/twingly-search-api-java)
 
-Java API for [Twingly Search](http://developer.twingly.com/resources/search/) (previously known as Twingly Analytics).
+Java API for [Twingly Search](https://developer.twingly.com/resources/search/) (previously known as Twingly Analytics).
 
 To use the API you need an API key. To obtain an API key, contact us at sales@twingly.com.
 
-### Example usage
+## Example usage
 
 ```Java
- Query q = new Query(yourApiKey);                // create a request object using your api key
- q.setDocumentLang("en");                        // only search for posts in english
- Result result = q.makeRequest("spotify");       // find posts containing the word spotify
- for (Post p : result.getPosts())
-     System.out.println(p.getUrl());             // print the url for each post
+// create Query object using your API key
```

Comment:
It's not full example, while I'll provide one later today, so I think there is no need for such imports and main method in readme file.